srs_data_management/generate_baseline_report.py

```diff
@@ -25,22 +28,30 @@ def checkout_model_repos(self, release):
         gmleng = Repo(constants.ASAPP_MLENG_ROOT)
         gmleng.git.checkout(release)
 
-        gcomcast = Repo(constants.ASAPP_COMCAST_SRS_ROOT)
-        gcomcast.git.checkout(release)
+        gclient = None
+        if self._client == 'condor':
+            gclient = Repo(constants.ASAPP_COMCAST_SRS_ROOT)
+        elif self._client == 'spear':
+            gclient = Repo(constants.ASAPP_SPRINT_SRS_ROOT)
+        gclient.git.checkout(release)
 
     def download_and_query_model(self, release):
         script =  os.path.join(os.path.dirname(os.path.realpath(__file__)), 'download_and_query_model.sh')
-        subprocess.run([script, release, self._baseline])
+        subprocess.run([script, self._client, release, self._baseline])
+
+    def query_historic_ami(self, release):
+        # TODO: query using the historic AMI tool
+        pass
```

Comment:
Could we put `NotImplementedError`? Will save the user a minute of trouble in debugging this.

Reply:
yep, good point